### PR TITLE
BSM stream processor/decoder refactor tracking

### PR DESCRIPTION
**Pull request by mvs5465**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -40,43 +40,57 @@ services:
...
-#     - kafka
-#    links:
-#     - kafka
+  ppm:
+    build: ../jpo-cvdp
+    environment:
+      DOCKER_HOST_IP: ${DOCKER_HOST_IP}
+    volumes:
+      - ${DOCKER_SHARED_VOLUME}:/ppm_data
+  s3dep:
```
Comment (hmusavi) [inline docker-compose.yml]: Shouldn't this container be called something like "cvpep_bsm_s3dep"? By the same token, property names and environmental variables should also bear the type of data being deposited.

File: docker-compose.yml [hunk 2]
```diff
@@ -40,43 +40,57 @@ services:
...
+      DOCKER_HOST_IP: ${DOCKER_HOST_IP}
+      DEPOSIT_BUCKET_NAME: ${CVPEP_DEPOSIT_BUCKET_NAME}
+      DEPOSIT_KEY_NAME: ${CVPEP_DEPOSIT_KEY_NAME}
+      DEPOSIT_TOPIC: ${CVPEP_DEPOSIT_TOPIC}
+    depends_on:
+     - kafka
+    links:
+     - kafka
+
+  ppms3dep:
```
Comment (hmusavi) [inline docker-compose.yml]: Shouldn't this container be called something like "rde_bsm_s3dep"? By the same token, property names and environmental variables should also bear the type of data being deposited.

File: docker-compose.yml [hunk 3]
```diff
@@ -40,43 +40,57 @@ services:
...
+      AWS_SECRET_ACCESS_KEY: ${PPM_AWS_SECRET_ACCESS_KEY_ID}
+      DOCKER_HOST_IP: ${DOCKER_HOST_IP}
+      DEPOSIT_BUCKET_NAME: ${PPM_DEPOSIT_BUCKET_NAME}
+      DEPOSIT_KEY_NAME: ${PPM_DEPOSIT_KEY_NAME}
+      DEPOSIT_TOPIC: ${PPM_DEPOSIT_TOPIC}
+    depends_on:
+     - kafka
+    links:
+     - kafka
+timdep:
```
Comment (hmusavi) [inline docker-compose.yml]: Shouldn't this container be called something like "cvpep_tim_s3dep"? By the same token, property names and environmental variables should also bear the type of data being deposited.

Comment (mvs5465) [reply docker-compose.yml]: Done.

**Comment by mvs5465** [comment]
Woops, now they should be.

**Pull request merged**